src/CMakeLists.txt

```diff
@@ -637,9 +637,23 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+
+set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
+if(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
+  set(ROCKSDB_EXTRA_CXXFLAG ${ROCKSDB_EXTRA_CXXFLAG} -DNDEBUG)
+endif(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
+
+# get number of CPUs to build rockdb with
+if(LINUX) 
```

Comment:
@tchaikov 
I looked at Cmake for RocksDB and it is pretty Windows targetted, so some stiff work ahead.
What do you suggest for the NPROC for RocksDB? This is how it is in Automake.